### PR TITLE
add urlconvention configuration

### PR DESCRIPTION
**Pull request by VikingsFan**

### CODE REVIEW

File: OData/src/System.Web.OData/Extensions/HttpConfigurationExtensions.cs [hunk 1]
```diff
@@ -288,6 +291,21 @@ public static void EnableNullDynamicProperty(this HttpConfiguration configuratio
         }
 
         /// <summary>
+        /// Set the UrlConventions in DefaultODataPathHandler.
```
Comment (xuzhg) [inline OData/src/System.Web.OData/Extensions/HttpConfigurationExtensions.cs]: comments?

File: OData/src/System.Web.OData/Extensions/HttpConfigurationExtensions.cs [hunk 2]
```diff
@@ -42,6 +43,8 @@ public static class HttpConfigurationExtensions
         private const string DoNotSerializeNullCollectionsSettingsKey =
             "System.Web.OData.DoNotSerializeNullCollectionsSettingsKey";
 
+        private static ODataUrlConventions urlConventions;
```
Comment (xuzhg) [inline OData/src/System.Web.OData/Extensions/HttpConfigurationExtensions.cs]: merge it into ResolverSetting

File: OData/test/UnitTest/System.Web.OData.Test/packages.config [hunk 3]
```diff
@@ -4,9 +4,9 @@
   <package id="Microsoft.AspNet.WebApi.Client" version="5.2.2" targetFramework="net45" />
   <package id="Microsoft.AspNet.WebApi.Core" version="5.2.2" targetFramework="net45" />
   <package id="Microsoft.AspNet.WebApi.WebHost" version="5.2.2" targetFramework="net45" />
-  <package id="Microsoft.OData.Core" version="6.13.0" targetFramework="net45" />
-  <package id="Microsoft.OData.Edm" version="6.13.0" targetFramework="net45" />
-  <package id="Microsoft.Spatial" version="6.13.0" targetFramework="net45" />
+  <package id="Microsoft.OData.Core" version="6.14.0" targetFramework="net45" />
```
Comment (xuzhg) [inline OData/test/UnitTest/System.Web.OData.Test/packages.config]: how about the e2e project?

**Comment by xuzhg** [comment]
looks good to me. Thanks.

**Comment by VikingsFan** [comment]
Merged 33c370b79fe055a57fbd6e3767a007437f2f8c41

**Pull request closed without merging**